packages/internals/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -800,4 +832,30 @@ describe('getGenerators', () => {
     expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
   })
+
+  test('fail if generator not found', async () => {
+    const aliases = {
+      'predefined-generator-1': {
+        generatorPath: generatorPath,
+        outputPath: __dirname,
+      },
+      'predefined-generator-2': {
+        generatorPath: generatorPath,
+        outputPath: __dirname,
+      },
+      'predefined-generator-3': {
+        generatorPath: generatorPath,
+        outputPath: __dirname,
+      },
+    }
+
+    await expect(
+      getGenerators({
+        schemaPath: path.join(__dirname, 'multiple-generators-schema.prisma'),
+        dataProxy: false,
+        providerAliases: aliases,
+        generatorNames: ['predefined-generator-1', 'invalid_generator'],
+      }),
+    ).rejects.toThrow('do not exist')
```

Comment:
Could we match the whole error message here?

If I understand correctly, only `invalid_generator` does not exist, so the error should have bee in singular, shouldn't it?

Reply:
Good catch, the test was wrong. Fixed in 4d150a3 :bow: